src/scripting/Scripting.cpp

```diff
@@ -599,11 +609,45 @@ void Scripting::TriggerOnOverlayClose() const
     m_store.TriggerOnOverlayClose();
 }
 
+sol::object Scripting::GetModNames() const
+{
+    auto lockedState = m_lua.Lock();
+    sol::table res(lockedState.Get(), sol::create);
+    for (const auto& name : m_store.GetModNames())
+        res.add(name);
+    return res;
+}
+
 sol::object Scripting::GetMod(const std::string& acName) const
 {
     return m_store.GetMod(acName);
 }
 
+sol::object Scripting::GetModInputHandlers(const std::string& acName) const
```

Comment:
Bindings can be hotkeys (up to 4 key combo, they activate always on key up) or inputs (single key, is called on key down/up). I dont see you use or work with this info. Also, I suppose this should return whether it is one or the other.

Reply:
Actually, if you were to do the GetMod change, I suppose we could also return handlers directly in it...

Also thinking now if it wouldnt make sense to introducd breaking change and make GetMod return single item from what GetMods would return, further simplifying the usage of this (table with mod info, bindings and return value from init.lua of the mod)

Reply:
Great, thanks! Now to just move the inner loop into ScriptContext and make GetMod also use it and I think this will be as clean as it gets.